Resources/public/js/views/tabs/ez-locationviewrelationstabview.js

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-locationviewrelationstabview', function (Y) {
+    "use strict";
+    /**
+     * Provides the Location View Relations Tab view class.
+     *
+     * @module ez-locationviewrelationsstabview
+     */
+    Y.namespace('eZ');
+
+    /**
+     * The Location View Relations tab class.
+     *
+     * @namespace eZ
+     * @class LocationViewRelationsTabView
+     * @constructor
+     * @extends eZ.LocationViewTabView
+     */
+    Y.eZ.LocationViewRelationsTabView = Y.Base.create('locationViewRelationsTabView', Y.eZ.LocationViewTabView, [Y.eZ.AsynchronousView], {
+        initializer: function () {
+            this._fireMethod = this._fireLoadObjectRelations;
+            this._watchAttribute = 'relatedContents';
+        },
+
+        render: function () {
+            var container = this.get('container');
+
+            container.setHTML(this.template({
+                "relatedContents": this._lookupRelationListItems(this.get('relatedContents')),
+                "loadingError": this.get('loadingError'),
+            }));
+
+            return this;
+        },
+
+        /**
+         * Provides the name of the relation type based on its Id
+         *
+         * @method _lookupRelationTypeName
+         * @protected
+         * @param {String} relationTypeId
+         * @return {String}
+         */
+        _lookupRelationTypeName: function(relationTypeId) {
+            if (relationTypeId === "ATTRIBUTE") {
+                return "Attribute";
+            }else if (relationTypeId === "EMBED") {
+                return "Embed";
+            }else if (relationTypeId === "COMMON") {
+                return "Content level relation";
+            }else {
+                return "Unknown relation type";
+            }
+        },
+
+        /**
+         * Provides the relation information for a given content
+         *
+         * @method _lookupRelationInfo
+         * @protected
+         * @param {eZ.Content} content
+         * @return {Array} of Relations struct:
+         *              struct.relationTypeName: Ready to be displayed name of the relation type
+         *              struct.fieldDefinitionName: Name of the field definition if any ("" if none)
+         */
+        _lookupRelationInfo: function(content) {
+            var relationInfo = [],
+                that = this,
+                contentType = this.get('contentType'),
+                fieldDefinitions = contentType.get('fieldDefinitions'),
+                fieldDefinitionName;
+
+            Y.Array.each(this.get('content').get('relations'), function (relation) {
+                if (content.get('id') === relation.destination) {
+                    fieldDefinitionName = "";
```

Comment:
a better default value would be to let that undefined (btw the var should be declared in the each function) as it does not mean anything in other relation than attributes.